sdk/sdkclients/sdkdeploymentsclient/client.go

```diff
@@ -20,25 +20,26 @@ type client struct {
 	client deploymentsv1connect.DeploymentsServiceClient
 }
 
-// Get implements sdkservices.Deployments.
-func (c *client) Get(ctx context.Context, id sdktypes.DeploymentID) (sdktypes.Deployment, error) {
-	resp, err := c.client.Get(ctx, connect.NewRequest(&deploymentsv1.GetRequest{DeploymentId: id.String()}))
+func New(p sdkclient.Params) sdkservices.Deployments {
+	return &client{client: internal.New(deploymentsv1connect.NewDeploymentsServiceClient, p)}
+}
+
+// Create implements sdkservices.Deployments.
+func (c *client) Create(ctx context.Context, deployment sdktypes.Deployment) (sdktypes.DeploymentID, error) {
```

Comment:
did we change the API here intentionally to return just the id and not the whole model?

Reply:
I don't think I've changed any API here. My diff shows me that Create was returning ID. Am I wrong?

Reply:
you are right, 
I read it wrong